js/apps/admin-ui/src/permissions-configuration/resource-types/ResourceType.tsx

```diff
@@ -95,7 +95,9 @@ export const ResourceType = ({
             resourceType: normalizedResourceType,
           })}
           defaultValue={[]}
-          variant="typeaheadMulti"
+          variant={withEnforceAccessTo ? "typeaheadMultiple" : "typeahead"}
```

Comment:
Did you mean `typeaheadMulti` instead of `typeaheadMultiple`?

Reply:
yes, good catch